money/money.py

```diff
@@ -1,16 +1,39 @@
 """Class representing a monetary amount"""
 
 from typing import Union
-from decimal import Decimal, ROUND_HALF_UP
+from decimal import Decimal
+from decimal import ROUND_CEILING
+from decimal import ROUND_DOWN
+from decimal import ROUND_FLOOR
+from decimal import ROUND_HALF_DOWN
+from decimal import ROUND_HALF_EVEN
+from decimal import ROUND_HALF_UP
+from decimal import ROUND_UP
+from decimal import ROUND_05UP
 from babel.numbers import format_currency
 from money.currency import Currency
 from money.currency import CurrencyHelper
 from money.exceptions import InvalidAmountError, CurrencyMismatchError, InvalidOperandError
 
+# pylint: disable=invalid-name
+DecimalRoundingType = Union[
```

Comment:
RoundingMode may be a more clear name, I think that's how python docs refer to it